### ISSUE TITLE
Add option 'allow-0rtt' for a bind of a frontend

### ISSUE THREAD

**Issue by olafnorge** [haproxytech/dataplaneapi#28]
Hi,

I am stuck at adding a ssl bind line like the following to a frontend:

```
bind *:443 name http_443 ssl crt /etc/haproxy/certs/ alpn h2,http/1.1 allow-0rtt
```

I am not able to set the `alpn` and the `allow-0rtt`.

Am I doing something wrong or oversee something? May you give me a tip  on how to achieve that?

**Comment by olafnorge** [comment]
Also pointing a folder to `crt` ends in an error. I wrote it here already: https://github.com/haproxytech/dataplaneapi/issues/27#issuecomment-533826671

**Comment by olafnorge** [comment]
Alright, the `alpn` is a undocumented field. I adapted my client and it works. Please update the open spec so that people see this field.

**But** I could not find an option to enable `allow-0rtt`. Are there plans to include it in the API?

**Comment by olafnorge** [comment]
> What do you mean undocumented? It is in the specification: https://github.com/haproxytech/dataplaneapi/blob/master/embedded_spec.go#L6033
> 
> We will add allow-0rtt in the future version, thank you for your feedback.

The `alpn` option is not showing up here: https://www.haproxy.com/documentation/dataplaneapi/latest/ which is the documentation I am following.

**Comment by mjuraga** [comment]
Thank you for pointing that, out we will fix it.

**Comment by mjuraga** [comment]
released in v1.2.4

**Comment by olafnorge** [comment]
Gracias
